apps/server/src/modules/tool/launch-tool/controller/api-test/tool-launch.controller.api.spec.ts

```diff
@@ -26,13 +25,14 @@ import {
 	schoolFactory,
 	userFactory,
 } from '@shared/testing';
-import request, { Response } from 'supertest';
-import { JwtAuthGuard } from '@src/modules/authentication/guard/jwt-auth.guard';
 import { ICurrentUser } from '@src/modules/authentication';
+import { JwtAuthGuard } from '@src/modules/authentication/guard/jwt-auth.guard';
```

Comment:
I think about the export of the jwt-auth.guard. But it is not nessasary for production and in test with newer testHelpers for the api test. If the test refactored. This import can be removed. It must not done in this PR.
But please note this as todo for refactoring for your team.